mlflow/tensorflow/__init__.py

```diff
@@ -515,7 +516,12 @@ def predict(self, data):
                 val = data[df_col_name]
                 if isinstance(val, pandas.DataFrame):
                     val = val.values
+
+                val = val.to_list()
```

Comment:
I don't think this works in the case where `isinstance(val, pandas.DataFrame)` is `True` because `val.values` is a numpy array and doesn't have a `to_list()` attribute.